src/tests/cptests/Makefile

```diff
@@ -14,13 +14,13 @@ build-tests: cptests
 run-tests: cptests
 	./cptests
 
-cptests: cptests.o $(parent_objs) $(parent_test_objs)
-	$(CXX) -o cptests cptests.o $(parent_test_objects) $(parent_objs) $(ALL_TEST_LDFLAGS)
+# Use objects from tests/ (../) directory because we already have them
+cptests_objs = $(foreach obj,$(parent_objs),../$(obj))
 
-$(objects): %.o: %.cc
-	$(CXX) $(ALL_TEST_CXXFLAGS) -MMD -MP -I../test-includes -I../../../dasynq/include -I../../../build/includes -I../../includes -c $< -o $@
+cptests: cptests.o $(cptests_objs) $(parent_test_objs)
```

Comment:
`$(parent_test_objs)` should be `$(parent_test_objects)` (it's a pre-existing mistake, but as you are changing the line then may as well fix it)